dumpling-overview.md

```diff
@@ -233,9 +233,18 @@ export AWS_ACCESS_KEY_ID=${AccessKey}
 export AWS_SECRET_ACCESS_KEY=${SecretKey}
 ```
 
-Dumpling also supports reading credential files from `~/.aws/credentials`. For more information about URI parameter descriptions, see [URI Formats of External Storage Services](/external-storage-uri.md).
+Dumpling also supports reading credential files from `~/.aws/credentials` and authenticating with AWS profiles. For more information about URI parameter descriptions, see [URI Formats of External Storage Services](/external-storage-uri.md).
+
+To use the default credentials from `~/.aws/credentials`:
+
+```shell
+tiup dumpling -u root -P 4000 -h 127.0.0.1 -r 200000 -o "s3://${Bucket}/${Folder}"
+```
+
+To use a specific AWS profile:
 
 ```shell
+export AWS_PROFILE=my-profile
```

Comment:
as in the unit test of https://github.com/pingcap/tidb/pull/62199

I see we should specify profile as `s3://bucket/prefix/?profile=staging`, not the style of environment variables

Reply:
I'm really sorry, I fixed to use the option that  I added in the 
https://github.com/pingcap/tidb/pull/62199

Reply:
https://github.com/pingcap/docs/pull/21456/commits/870cc2c5e145221d8d552a81ced42b116374cf1d